src/src/WebServer/Markup.cpp

```diff
@@ -1010,6 +1010,14 @@ void addRTDControllerButton(cpluginID_t cpluginID) {
       F("Controller/%s.html"),
       get_formatted_Controller_number(cpluginID).c_str()));
 }
+
+void   addRTDNetworkDriverButton(ESPEasy::net::nwpluginID_t nwpluginID)
+{
+  addRTDHelpButton(
+    strformat(
+      F("NetworkDriver/%s.html"),
```

Comment:
Shouldn't that be `Network` instead of `NetworkDriver` ? The docs folder is named `Network`, AFAICS 🤔